io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/writer/AbstractTableInstanceWriter.java

```diff
@@ -79,45 +90,30 @@ public boolean isCancelable() {
 				if (solveNestedProperties && property instanceof Group) {
 					Group nextInstance = (Group) property;
 					iterateBuild(nextInstance, qname, headerRow, row, cellValue);
-//					Group inst = (Group) property;
-//					// check if property has a value and add it
-//					checkValue(inst, headerRow, row, cellValue);
-//					// go through nested properties to get other properties
-//					Iterator<QName> propertyIt = inst.getPropertyNames().iterator();
-//					if (propertyIt.hasNext()) {
-//						QName value = propertyIt.next();
-//						Object nextProp = inst.getProperty(value)[0];
-//						// check if current property should be displayed in map
-//						if (shouldBeDisplayed(nextProp)) {
-//							cellValue += ".";
-//							cellValue += value.getLocalPart();
-//						}
-//
-//						// iterate over all nested properties
-//						while (nextProp instanceof Group) {
-//							Group oinst = (Group) nextProp;
-//							checkValue(oinst, headerRow, row, cellValue);
-//
-//							// get localparts of all nested properties
-//							Iterator<QName> qnames = oinst.getPropertyNames().iterator();
-//							if (qnames.hasNext()) {
-//								value = qnames.next();
-//								nextProp = oinst.getProperty(value)[0];
-//								if (shouldBeDisplayed(nextProp)) {
-//									cellValue += ".";
-//									cellValue += value.getLocalPart();
-//								}
-//								else
-//									continue;
-//							}
-//							else
-//								break;
-//						}
-//						// add property with corresponding cellValue (localpart)
-//						// to map
-//						// no resolving of nested properties
-//						addProperty(headerRow, row, nextProp, cellValue);
-//					}
+				}
+				else {
+					// add property with corresponding cellValue (localpart) to
+					// map
+					if (property instanceof Group && shouldBeDisplayed(property)) {
+						checkValue((Group) property, headerRow, row, cellValue);
+					}
+					else {
+						addProperty(headerRow, row, property, cellValue);
+					}
+
+				}
+			}
+			else {
```

Comment:
To retain the previous behavior in case `useSchema` is not enabled, shouldn't this be `else if (useSchema)` then?